From 79995bc82e81f0429222aab4d90a03440feef057 Mon Sep 17 00:00:00 2001
From: Matan Ziv-Av <matan@svgalib.org>
Date: Mon, 29 Jun 2020 15:18:03 +0300
Subject: [PATCH] Use base64.decodebytes.

Based on upstream commit 79995bc82e81f0429222aab4d90a03440feef057 which
confuses several unrelated changes.

---
 tremc | 29 ++++++++++++++++++++++++++++-
 1 file changed, 28 insertions(+), 1 deletion(-)

diff --git a/tremc b/tremc
index b47f012..c626fde 100755
--- a/tremc
+++ b/tremc
@@ -440,7 +443,7 @@ class Transmission(object):
                 # TAG_TORRENT_DETAILS, but just passing seems to help.(?)
                 try:
                     torrent_details = response['arguments']['torrents'][0]
-                    torrent_details['pieces'] = base64.decodestring(bytes(torrent_details['pieces'], ENCODING))
+                    torrent_details['pieces'] = base64.decodebytes(bytes(torrent_details['pieces'], ENCODING))
                     self.torrent_details_cache = torrent_details
                     self.upgrade_peerlist()
                 except IndexError:
